### ISSUE TITLE
Empty get shows an error

### ISSUE THREAD

**Issue by toonvd** [jhthorsen/mojo-redis#35]
Hi

I am working in a high concurrency environment where first doing an exists before a get, hits too hard on performance.

Hence why I assign a get command straight to a variable.
Doing this gives me the following error message:

Use of uninitialized value $bytes in string at /home/XXX/perl5/lib/perl5/Mojo/Util.pm line 121.
That function is trying a decode on an empty value.
```
return undef
    unless eval { $bytes = _encoding($encoding)->decode("$bytes", 1); 1 };
```

Example code:
```
my $redis_instance = Mojo::Redis->new( 'redis://' . $redis_cache_host );
$redis_instance->protocol_class('Protocol::Redis::XS');

my $redis= $redis_instance->db;

my $test = $redis->get('namespace:key');
```

**Comment by Grinnz** [comment]
What version of Mojo::Redis are you using? This sounds like https://github.com/jhthorsen/mojo-redis/pull/19 which was fixed in 3.08.

**Comment by toonvd** [comment]
Hi

You are completely right, I forgot to install my snapshot and an older version from my global lib was loaded :|.

Thanks for the hint! And nice lib all!
